joomla-projects/privacy-framework
# System - Log Rotation plugin

Pull Request for Issue # .

### Summary of Changes
This is a replacement for https://github.com/joomla-projects/privacy-framework/pull/51 . The original PR now could not be merged because a mistake in my PR somehow. Sorry @alikon for stealing your work.

### Testing Instructions
Install a copy of my branch, test and make sure log files are rotated and deleted properly. Ping @sandewt for human testing as he tested the original PR and pointed out a potential issue
